cmd/kubeadm/app/util/apiclient/idempotency_test.go

```diff
@@ -139,7 +939,7 @@ func TestPatchNode(t *testing.T) {
 				}
 			}, &lastError)
 			success, err := conditionFunction(context.Background())
-			if err != nil {
+			if err != nil && success {
```

Comment:
`tc.success` is more readable to me.

Reply:
yes that's better. updated
https://github.com/kubernetes/kubernetes/compare/2941a644905755c4a89f73024666dc50da471ad2..c29450eb0090e53db16c37a6468a57f6b916f827